packages/typespec-test/test/widget_dpg/generated/typespec-ts/sources/generated/src/api/operations.ts

```diff
@@ -55,7 +55,7 @@ export async function _listWidgetsDeserialize(
   return (result.body ?? []).map((p) => ({
     id: p["id"],
     weight: p["weight"],
-    color: p["color"],
+    color: p["color"] as any,
```

Comment:
In the latest tcgc this is detected as combine type for `"a" | "b"` before it is just a named enum. So we take it as any for union.
https://github.com/Azure/autorest.typescript/blob/upgrade-to-v0.49/packages/typespec-ts/src/modular/helpers/operationHelpers.ts#L818